ast/position.go

```diff
@@ -18,9 +18,17 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package idl
+package ast
 
 // Position represents a position in the parsed document.
 type Position struct {
 	Line int
 }
+
+// Pos attempts to return the position of a Node in the parsed document.
+func Pos(n Node) (Position, bool) {
```

Comment:
Worth pointing to Info here?


```suggestion
// Pos attempts to return the position of a Node in the parsed document.
// For most use cases, prefer to use idl.Info to access positional information.
func Pos(n Node) (Position, bool) {
```